app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirestoreTaskTest.java

```diff
@@ -0,0 +1,200 @@
+package com.github.houseorganizer.houseorganizer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.github.houseorganizer.houseorganizer.task.FirestoreTask;
+import com.github.houseorganizer.houseorganizer.task.Task;
+import com.github.houseorganizer.houseorganizer.user.DummyUser;
+import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Before this test class, the firestore emulator is enabled.
+ * Before each test, a test task list is created, and it is wiped out after each test.
+ *
+ * At the moment, doesn't use household or user data.
+ */
+@RunWith(AndroidJUnit4.class)
+public class FirestoreTaskTest {
+    public static final String BASIC_SUBTASK_TITLE = "Subtask";
+    public static final String STATUS_ONGOING = "ongoing";
+    public static final String STATUS_COMPLETED = "completed";
+    public static final String BASIC_TASK_TITLE = "Task1";
+    public static final String BASIC_TASK_NAME = "Desc1";
+    public static final String NEW_FANCY_TITLE = "New fancy title";
+    public static final String NEW_FANCY_DESCRIPTION = "New fancy description";
```

Comment:
If these are useful elsewhere maybe they have a place in the firebase helper, not sure about this one

Reply:
I agree with you, and will probably move them as I start testing other task list elements